### PR TITLE
Improved Multi-Org tutorial, added ACL tutorial to docs

### PR DESCRIPTION
**Pull request by mahoney1**
Improve the flow of the multi org tutorial and add a new ACL tutorial

Signed-off-by: Paul O'M <mahoney@uk.ibm.com>
## Checklist
 - [ ]  A link to the issue/user story that the pull request relates to
#3148
#2424 (for tutorial, rather than full guide)
 - [ ]  How to recreate the problem without the fix
 - [ ]  Design of the fix
 - [ ]  How to prove that the fix works
 - [ ]  Automated tests that prove the fix keeps on working
 - [ ]  Documentation - any JSDoc, website, or Stackoverflow answers?

## Issue/User story

#3148
#2424 (for tutorial, rather than guide)

## Steps to Reproduce

1.
2.
3.
4.

## Existing issues

- [ ] [Stack Overflow issues](http://stackoverflow.com/tags/hyperledger-composer)
- [ ] [GitHub Issues](https://github.com/hyperledger/composer/issues)
- [ ] [Rocket Chat history](https://chat.hyperledger.org/channel/composer)

### CODE REVIEW

File: packages/composer-website/jekylldocs/tutorials/deploy-to-fabric-multi-org.md [hunk 1]
```diff
@@ -33,30 +39,45 @@ Let's get started!
 
 <h2 class='everybody'>Prerequisites</h2>
 
-If you have installed the development environment, you will need to first stop the {{site.data.conrefs.hlf_full}} provided by the development environment:
+If you have previously installed the Composer IDE / development environment, you will need to first tear down the {{site.data.conrefs.hlf_full}} containers provided by the development environment:
```
Comment (davidkel) [inline packages/composer-website/jekylldocs/tutorials/deploy-to-fabric-multi-org.md]: I don't think we should be referring to a Composer IDE. Playground isn't an IDE and VSCode plugin doesn't need to be mentioned here.

File: packages/composer-website/jekylldocs/tutorials/deploy-to-fabric-multi-org.md [hunk 2]
```diff
@@ -33,30 +39,45 @@ Let's get started!
...
 
     cd ~/fabric-tools
     ./stopFabric.sh
     ./teardownFabric.sh
 
-Clone the following GitHub repository:
+Next, clone the following GitHub Fabric Samples repository (rather than use the samples from the [Fabric site:](http://hyperledger-fabric.readthedocs.io/en/latest/build_network.html) at this point, it needs some changes)
```
Comment (davidkel) [inline packages/composer-website/jekylldocs/tutorials/deploy-to-fabric-multi-org.md]: "it needs some changes". Not sure what that is telling me, do I need to make changes for example ? what changes ? etc

File: packages/composer-website/jekylldocs/tutorials/deploy-to-fabric-multi-org.md [hunk 3]
```diff
@@ -33,30 +39,45 @@ Let's get started!
...
+We are using the  [Building Your First Network](http://hyperledger-fabric.readthedocs.io/en/latest/build_network.html) Fabric sample network for this multi-org tutorial. We will refer to this {{site.data.conrefs.hlf_full}} network as the 'BYFN' (Building Your First Network) network henceforth. If you choose to split your organisations across separate physical machines or separate virtual machines running on different IP networks, it is outside the scope of this particular tutorial. You will need to configure the blockchain network (for each org) and check that you can deploy/interact with some Fabric chaincode samples deployed to that network. Furthermore, the container names defined below in the connection profiles are resolved easily on single-machine based tutorial - you will need to ensure your setup is configured for any resolution or routing that is required between the machines.
```
Comment (davidkel) [inline packages/composer-website/jekylldocs/tutorials/deploy-to-fabric-multi-org.md]: Think we can stop at the point saying this is outside the scope of this tutorial.

File: packages/composer-website/jekylldocs/tutorials/deploy-to-fabric-multi-org.md [hunk 4]
```diff
@@ -33,30 +39,45 @@ Let's get started!
...
+    curl -sSL https://goo.gl/6wtTN5 | bash -s 1.1.0-preview
 
-You can now start the BYFN network. You must specify additional flags that are not specified in the Building Your First Network tutorial. This is because we want to use CouchDB as the world state database, and we want to start a Certificate Authority (CA) for each organization.
+    Verify the list of docker images downloaded without issues
+    
+3. change directory into `first-network` sample 
+
+    cd first-network
+
+4. Next, start the BYFN network - additional flags (to the `byfn.sh` script below) must be specified, as we're using CouchDB as the world state database (different to that specified on the Fabric BYFN page) - we also want to start a Certificate Authority (CA) for **each** organization.
```
Comment (davidkel) [inline packages/composer-website/jekylldocs/tutorials/deploy-to-fabric-multi-org.md]: There are 2 steps to byfn, we should explain the generate step if we are going to provide information about byfn itself.

File: packages/composer-website/jekylldocs/tutorials/deploy-to-fabric-multi-org.md [hunk 5]
```diff
@@ -33,30 +39,45 @@ Let's get started!
...
+ 
+5. Execute the following commands in sequence from the `first-network` directory:
 
     ./byfn.sh -m generate
+    
     ./byfn.sh -m up -s couchdb -a
 
-If the command works successfully, the BYFN network is started, and you will see the following output:
+If the command works successfully, the first command will generate artifacts, then following the second command, the BYFN network is started, and verify that you see the following output before proceeding:
```
Comment (davidkel) [inline packages/composer-website/jekylldocs/tutorials/deploy-to-fabric-multi-org.md]: Given we are providing more info we should explain a bit more about the generate step or point back to the original byfn documentation.

File: packages/composer-website/jekylldocs/tutorials/deploy-to-fabric-multi-org.md [hunk 6]
```diff
@@ -281,43 +302,42 @@ We need a base connection profile that describes this fabric network which can t
         }
     }
 
-When working with the certificates it is recommended to create a temporary working directory to create the Composer connection profiles:
+Copy this base file (above) into a file  `byfn-network.json` under the new directory `/tmp/composer`. When working with the certificates, it is recommended to create a temporary working directory to create the Composer connection profiles.
```
Comment (davidkel) [inline packages/composer-website/jekylldocs/tutorials/deploy-to-fabric-multi-org.md]: /tmp/composer doesn't exist until you do the mkdir commands that follow

File: packages/composer-website/jekylldocs/tutorials/deploy-to-fabric-multi-org.md [hunk 7]
```diff
@@ -281,43 +302,42 @@ We need a base connection profile that describes this fabric network which can t
...
     mkdir -p /tmp/composer/org2
 
-You need to replace all instances of the text `INSERT_ORG1_CA_CERT` with the CA certificate for the peer nodes for `Org1`:
-Use the following command to convert the pem file to something that can be embedded into the above connection profile.
+Open `byfn-network.json` and replace all instances of the text `INSERT_ORG1_CA_CERT` with the CA certificate for the peer nodes for `Org1`: - use the following command to get the certificate from the .pem file so that it can be embedded into the above connection profile.
 
     awk 'NF {sub(/\r/, ""); printf "%s\\n",$0;}' crypto-config/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls/ca.crt > /tmp/composer/org1/ca-org1.txt
 
-copy the contents of the file `/tmp/composer/org1/ca-org1.txt` and replace the text `INSERT_ORG1_CA_CERT`. It should now look something like this (but be a single line in the profile file)
+Next, copy the contents of the file `/tmp/composer/org1/ca-org1.txt` and replace the text `INSERT_ORG1_CA_CERT` in the .json file. It should now look something like this (must be a single line in the profile file as shown)
```
Comment (davidkel) [inline packages/composer-website/jekylldocs/tutorials/deploy-to-fabric-multi-org.md]: Next implies there is a first step, so we should define a first step in the section above.

File: packages/composer-website/jekylldocs/tutorials/deploy-to-fabric-multi-org.md [hunk 8]
```diff
@@ -281,43 +302,42 @@ We need a base connection profile that describes this fabric network which can t
...
-You need to replace all instances of the text `INSERT_ORDERER_CA_CERT` with the CA certificate for the orderer node:
-Use the following command to convert the pem file to something that can be embedded into the above connection profile.
+Next, you need to replace all instances of the text `INSERT_ORDERER_CA_CERT` with the CA certificate for the orderer node:  use the following command to convert the .pem file to something that can be embedded into the above connection profile json file.
 
     awk 'NF {sub(/\r/, ""); printf "%s\\n",$0;}' crypto-config/ordererOrganizations/example.com/orderers/orderer.example.com/tls/ca.crt > /tmp/composer/ca-orderer.txt
 
 copy the contents of the file `/tmp/composer/ca-orderer.txt` and replace the text `INSERT_ORDERER_CA_CERT`.
 
-Once done, save this file as `/tmp/composer/byfn-network.json`.
+Once done, save this file in your edit sessions as  `/tmp/composer/byfn-network.json`.
```
Comment (davidkel) [inline packages/composer-website/jekylldocs/tutorials/deploy-to-fabric-multi-org.md]: I don't know what you mean by "in your edit sessions" so would suggest it be removed.

File: packages/composer-website/jekylldocs/tutorials/deploy-to-fabric-multi-org.md [hunk 9]
```diff
@@ -281,43 +302,42 @@ We need a base connection profile that describes this fabric network which can t
...
 
 This connection profile now describes the fabric network setup, all the peers, orderers and certificate authorities that are part of the network, it defines all the organizations that are participating in the network and also defines the channel's on this network. {{site.data.conrefs.composer_full}} can only interact with a single channel so only one channel should be defined.
 
 <h2 class='alice'>Step Three: Customizing the connection profile for Org1</h2>
 
-This is just a case of now specifying the organization that `alice` belongs to in a client section with optional timeouts, so add the following block into the above connection profile `/tmp/composer/byfn-network.json` after the `version` property and before the `channel` property and save it as `/tmp/composer/org1/byfn-network-org1.json`.
+This is just a case of specifying the organization that `alice` belongs to, in a `client` section with optional timeouts,  add the following block into the above connection profile `/tmp/composer/byfn-network.json` between `version` property and just before the `channel` property - once done, save it as a NEW file called  `/tmp/composer/org1/byfn-network-org1.json`.
```
Comment (davidkel) [inline packages/composer-website/jekylldocs/tutorials/deploy-to-fabric-multi-org.md]: between the version (need the)

File: packages/composer-website/jekylldocs/tutorials/deploy-to-fabric-multi-org.md [hunk 10]
```diff
@@ -375,17 +395,48 @@ Repeat the same process for `bob` but this time specify the organization as `Org
         "channel": {
         ...
 
+So the section of the profile should look like
```
Comment (davidkel) [inline packages/composer-website/jekylldocs/tutorials/deploy-to-fabric-multi-org.md]: Why is this section repeated again ? it looks like the example directly above it.

File: packages/composer-website/jekylldocs/tutorials/deploy-to-fabric-multi-org.md [hunk 11]
```diff
@@ -375,17 +395,48 @@ Repeat the same process for `bob` but this time specify the organization as `Org
...
+        ...
+
 <h2 class='alice'>Step Five: Locating the certificate and private key for the {{site.data.conrefs.hlf_full}} administrator for Org1</h2>
 
-The administrator for our {{site.data.conrefs.hlf_full}} network is a user called `Admin@org1.example.com`. The certificates and private key files for this user are stored in the directory:
+The administrator for our {{site.data.conrefs.hlf_full}} Org1 network is a user called `Admin@org1.example.com`. The certificates and private key files for this user are stored in the directory:
 
     crypto-config/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp
 
+You can set this directory as a variable to use for copying the certificate and key:
```
Comment (davidkel) [inline packages/composer-website/jekylldocs/tutorials/deploy-to-fabric-multi-org.md]: huh ?

File: packages/composer-website/jekylldocs/tutorials/deploy-to-fabric-multi-org.md [hunk 12]
```diff
@@ -439,13 +499,15 @@ If the command works successfully, a business network card called `PeerAdmin@byf
...
-    composer runtime install -c PeerAdmin@byfn-network-org1 -n tutorial-network
+    composer runtime install -c PeerAdmin@byfn-network-org1 -n trade-network
 
 <h2 class='bob'>Step Twelve: Installing the {{site.data.conrefs.composer_full}} runtime onto the {{site.data.conrefs.hlf_full}} peer nodes for Org2</h2>
 
 Run the `composer runtime install` command to install the {{site.data.conrefs.composer_full}} runtime onto all of the {{site.data.conrefs.hlf_full}} peer nodes for `Org2` that you specified in the connection profile file you created in step four:
 
-    composer runtime install -c PeerAdmin@byfn-network-org2 -n tutorial-network
+    composer runtime install -c PeerAdmin@byfn-network-org2 -n trade-network
+
```
Comment (davidkel) [inline packages/composer-website/jekylldocs/tutorials/deploy-to-fabric-multi-org.md]: Shouldn't this be at the first place where trade-network is referenced.

**Comment by mahoney1** [comment]
added further changes to multi-org tutorial after review and Signed-off-by: Paul O'M mahoney@uk.ibm.com

**Pull request closed without merging**